admin/model_cloud_provider_access_role.go

```diff
@@ -20,10 +20,10 @@ type CloudProviderAccessRole struct {
 	// Date and time when someone authorized this role for the specified cloud service provider. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
 	// Read only field.
 	AuthorizedDate *time.Time `json:"authorizedDate,omitempty"`
-	// Date and time when someone created this role for the specified cloud service provider. This parameter expresses its value in the ISO 8601 timestamp format in UTC.  Alternatively: Date and time when this Azure Service Principal was created. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
+	// Date and time when this Azure Service Principal was created. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
 	// Read only field.
 	CreatedDate *time.Time `json:"createdDate,omitempty"`
-	// List that contains application features associated with this Amazon Web Services (AWS) Identity and Access Management (IAM) role.  Alternatively: List that contains application features associated with this Azure Service Principal.
+	// List that contains application features associated with this Azure Service Principal.
```

Comment:
non blocking (just my understanding): why is the "alternatively" being removed? Was the previous code trying to merge two different documentations?

Reply:
Great question. Initial design had merged description from all possible types that API can handle. In practice descriptions were confusing and very long. I have settled with base class description which usually reflects better what field mean. In some cases that base class would require small correction but it is still much better than having +60 lines description that does not mean much.